samples/TextGeneration.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import GoogleGenerativeAI
+import XCTest
+
+@available(iOS 15.0, macOS 11.0, macCatalyst 15.0, *)
+final class TextGeneration: XCTestCase {
+  override func setUpWithError() throws {
+    try XCTSkipIf(APIKey.default.isEmpty, "`\(APIKey.apiKeyEnvVar)` environment variable not set.")
+  }
+
+  func testTextOnlyPrompt() async throws {
+    // [START text_gen_text_only_prompt]
+    let generativeModel =
+      GenerativeModel(
+        // Specify a Gemini model appropriate for your use case
+        name: "gemini-1.5-flash",
+        // Access your API key from your on-demand resource .plist file (see "Set up your API key"
```

Comment:
One comment I got from Rachel here was that this "see ... above" comment makes sense in the docs, but not if you are looking at this file, since there's nothing there. I ended up adding a comment block on each file.

See https://github.com/google-gemini/generative-ai-android/pull/182/commits/75317e98c5a449c0384f29cb37e246db5239142f

**That being said**, I haven't got any feedback on it yet.

Reply:
Good idea, I added a very similar one to yours (just not in Markdown since it won't get rendered by Xcode in a standard comment block).